eng/pipelines/publish.yml

```diff
@@ -69,6 +73,18 @@ jobs:
                 /p:NuGetApiKey=$(dotnet-myget-org-api-key)
         displayName: Push to myget.org
 
+      - script: powershell -ExecutionPolicy ByPass -NoProfile eng\common\msbuild.ps1 build.proj
+                -warnaserror:0 -ci
+                /t:UpdatePublishedVersions
+                /p:GitHubUser=dotnet-helix-bot
+                /p:GitHubEmail=dotnet-helix-bot@microsoft.com
```

Comment:
Wrong bot, should be `dotnet-build-bot` and `dotnet-build-bot@microsoft.com`. The values here probably came from the old build definition, but those defaults are overridden by the root pipebuild IIRC.

Reply:
Actually for these two guys, it doesn't seem like we override them: https://github.com/dotnet/corefx/blob/master/buildpipeline/DotNet-Trusted-Publish.json#L209 those are hardcoded in the command line parameters directly. 

Should we use those ones, or the ones you suggested?

Reply:
Interesting... it looks like they don't actually have any effect because the PAT is really all GitHub needs to identify the user, and the VersionTools code doesn't do anything special to force the resulting dotnet/versions commit to use this user/email.

I'd suggest removing these two props entirely.

Reply:
Yep, that seems like it. I will remove those as the task sets them to dotnet-bot when they are null. It seems like we use them for the committer info: https://github.com/dotnet/buildtools/blob/6736870b84e06b75e7df32bb84d442db1b2afa10/src/Microsoft.DotNet.VersionTools/Automation/GitHubApi/GitHubClient.cs#L126

Reply:
From looking at the commits in the versions repo, it seems like GH ignores that and just uses the PAT.

Reply:
Ah, forgot about that API--we actually switched to use [`PostCommitAsync`](https://github.com/dotnet/buildtools/blob/6736870b84e06b75e7df32bb84d442db1b2afa10/src/Microsoft.DotNet.VersionTools/Automation/GitHubApi/GitHubClient.cs#L329) rather than `PutGitHubFileAsync` so we could submit multiple files in a single commit. That's why it mattered before and doesn't anymore.